cylc_singleuser.py

```diff
@@ -125,6 +153,14 @@ def start(self):
         signal.signal(signal.SIGINT, app.signal_handler)
         app.listen(self._port)
         ioloop.PeriodicCallback(app.try_exit, 100).start()
+        # Discover workflows on intiial start up.
+        ioloop.IOLoop.current().add_callback(self.ws_mgr.gather_workflows)
+        # Arbitrary intervals (msec) chosen for the two periodic callbacks
+        # below. These will likely change or be removed with PUB/SUB updates.
+        # Check for new workflows every 20sec.
+        ioloop.PeriodicCallback(self.ws_mgr.gather_workflows, 20000).start()
+        ioloop.PeriodicCallback(
+            self.data_mgr.entire_workflow_update, 5000).start()
```

Comment:
Yeah I could stick to prime numbers (i.e 3 & 19)... If the callbacks take longer than the specified time Tornado won't run the next one (BTW). However (might be a way!), as it is, we can't always guarantee that they won't clash because we don't know how long they both take to complete.. So, bar running them synchronously, we might just have to catch the error in the data manager if/when it ever occurs.